litmus-portal/frontend/src/components/Sections/Settings/AccountsTab/AccountSettings/index.tsx

```diff
@@ -139,99 +117,34 @@ const AccountSettings: React.FC = () => {
             <form className={classes.innerPass}>
               {/* Current Password */}
-              <FormControl>
-                <Input
-                  className={classes.pass}
-                  defaultValue={currPassword.password}
-                  id="outlined-adornment-password"
-                  type={currPassword.showPassword ? 'text' : 'password'}
-                  disableUnderline
-                  endAdornment={
-                    <InputAdornment position="end">
-                      <IconButton
-                        aria-label="toggle password visibility"
-                        onClick={handleClickShowPassword1}
-                        edge="end"
-                      >
-                        {currPassword.showPassword ? (
-                          <Visibility />
-                        ) : (
-                          <VisibilityOff />
-                        )}
-                      </IconButton>
-                    </InputAdornment>
-                  }
-                />
-                <InputLabel htmlFor="outlined-adornment-password">
-                  Current Password
-                </InputLabel>
-              </FormControl>
+              <InputField
+                required
+                value={currPassword.password}
+                handleChange={handleCurrPassword('password')}
+                type="password"
+                label="Current Password"
+                validationError={currPassword.err}
+              />
+
               {/* New Password */}
-              <FormControl>
-                <Input
-                  data-cy="changePassword"
-                  className={`${classes.pass} ${
-                    newPassword.err ? classes.error : classes.success
-                  }`}
-                  id="outlined-adornment-password"
-                  type={newPassword.showPassword ? 'text' : 'password'}
-                  onChange={handleNewPassword('password')}
-                  disableUnderline
-                  endAdornment={
-                    <InputAdornment position="end">
-                      <IconButton
-                        data-cy="conVisibilty"
-                        aria-label="toggle password visibility"
-                        onClick={handleClickShowPassword2}
-                        onMouseDown={handleMouseDownPassword}
-                        edge="end"
-                      >
-                        {newPassword.showPassword ? (
-                          <Visibility data-cy="visIcon" />
-                        ) : (
-                          <VisibilityOff data-cy="invisIcon" />
-                        )}
-                      </IconButton>
-                    </InputAdornment>
-                  }
-                />
-                <InputLabel htmlFor="outlined-adornment-password">
-                  New Password
-                </InputLabel>
-              </FormControl>
+              <InputField
```

Comment:
Add a `success` field here, you can use validateConfirmPassword and negate it or use useRef. Take a look at `Stepper` password validation for reference.